public/css/showoff.css

```diff
@@ -135,6 +135,11 @@ pre code {
   }
 }
 
+.ui-button:active {
```

Comment:
At this time, the only other place where .ui-button is being used is in the presenter topbar; these colors match the active and hover state of those buttons.